content/post/2020-10-06-survey-wave-4.Rmd

```diff
@@ -0,0 +1,264 @@
+and help contact tracers do their work of tracking down new cases. Shortages of
+tests can impede contact tracing and can also be harmful in settings like
+nursing homes, where many residents are particularly vulnerable to COVID-19 and
+staff must be tested regularly to ensure residents aren't exposed.
+
+Testing data is published by state health authorities and aggregated by
+organizations such as the [COVID Tracking Project](https://covidtracking.com/);
+our own [COVIDcast map](https://covidcast.cmu.edu/) also includes data from
+antigen tests produced by Quidel, a major test product manufacturer. But a
+survey allows us to ask people directly whether they have been tested, and
+allows us to understand *who* has been tested.
+
+For example, we can ask: What percentage of people in each state say they were
+tested in the last 14 days?
+
+```{r pct-tested, message=FALSE}
+tested <- covidcast_signal("fb-survey", "wip_smoothed_wtested_14d",
+                           start_day = end_day, end_day = end_day,
+                           geo_type = "state")
+plot(tested, range = c(6, 25),
+     title = "% tested in the last 14 days") +
+    plot_label + date_label
+```
+
+We can see a strong variation across the country---in North Dakota, roughly a
+quarter of people were tested in the last 14 days!
+
+It's worth thinking about this result in more detail. If we take North Dakota's
+adult population, 25% of that population is roughly twice as many tests as North
+Dakota's publicly reported testing data says were conducted during that time
+period. But North Dakota's published testing figures [only include PCR tests,
+not COVID antigen
+tests](https://khn.org/news/lack-of-antigen-test-reporting-leaves-country-blind-to-the-pandemic/),
+meaning the public figures leave out an important part of testing that the
+survey data can illuminate. Antigen tests, and new rapid tests, will only become
+more important in the coming months, making it important for epidemiologists to
+have information about their use.
+
+But testing volume is only one part of the issue. Another question is how many
+people are testing positive. If a large percentage of tests have positive
+results, that suggests that tests are only being given to people at high risk of
+illness, rather than being used in mass testing campaigns. When survey
+respondents say they were tested in the past 14 days, we also ask if they tested
+positive:
+
+```{r test-positivity, message=FALSE}
+positivity <- covidcast_signal("fb-survey", "wip_smoothed_wtested_positive_14d",
+                               start_day = end_day, end_day = end_day,
+                               geo_type = "state")
+plot(positivity, range = c(4, 22),
+     title = "% of tests that were positive, last 14 days") +
+    plot_label + date_label
+```
+
+We can see quite a large variation in test positivity across the United States,
+suggesting different access to testing.
```

Comment:
I didn't add Quidel since that starts getting more technical (difference in test types, they're only one manufacturer, etc.), but I clarified the language about test positivity